src/components/Toast/README.md

```diff
@@ -302,7 +362,7 @@ On iOS, icons are available for cases where you want to re-inforce the message.
 
 <!-- example-for: android, ios, web -->
 
-Use error toast to indicate that an operation has failed. For example, your phone is offline and need to reconnect to the internet. For all other error message types, follow the [error message guidelines](/patterns/error-messages).
+Although error toasts are still available and used in the system, we discourage their use. Reserve them for errors not caused by merchants, like a connection issue. Error toasts should convey what went wrong in plain language and should not go over 3 words. For all other error message types, follow the [error message guidelines](/patterns/error-messages).
```

Comment:
Changes to error were included in this PR because adding an action to an error toast modified the guidelines.